content/en/api/logs/code_snippets/list_logs.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# Make sure you replace the <DD_API_KEY> and <DD_APP_KEY> key below
+# with the ones for your account
+
+api_key=<DD_API_KEY>
+app_key=<DD_APP_KEY>
+
+curl -X GET https://http-intake.logs.datadoghq.com/api/v1/logs-queries/list?api_key=${api_key}&application_key=${app_key}
+     -H "Content-Type: application/json" \
+     -d '{query: "service:nginx query -method:POST Windows",time: {offset: -3600, from: "2001-01-01 12:00:00", to: "2001-01-01 14:00:00"} sort: "desc", limit: 50}'
```

Comment:
time must be something like `2019-03-02T09:42Z` or `2019-03-02T09:50:36.320Z` to work properly